src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -117,12 +118,21 @@ public KillBillHttpClient(final String kbServerUrl, final String username, final
         this.apiKey = apiKey;
         this.apiSecret = apiSecret;
 
-
         final AsyncHttpClientConfig.Builder cfg = new AsyncHttpClientConfig.Builder();
 
+        if(requestTimeout != null)
+        {
+            cfg.setRequestTimeout(requestTimeout);
+            requestTimeoutSec = (int) Math.ceil((double) requestTimeout / 1000);
```

Comment:
Maybe I am missing something, but all the methods expect the timeout coming in seconds. I could change the executeAndWait to expect ms, but then if the timeout was specified in the params of the methods it would be expected to be in ms instead of seconds. This seems like a big change.